requirements.txt

```diff
@@ -40,3 +40,4 @@ langchain-text-splitters
 # the below library should NOT be imported into any python files
 # it is for CLI usage ONLY
 yamllint>=1.35.1,<1.36.0
+prettytable
```

Comment:
It would be better to include this in the change that makes use of the dependency.

Also, it should have version guardrails similar to the rest of the entries.

Reply:
```ERROR: Ignored the following versions that require a different python version: 1.21.2 Requires-Python >=3.7,<3.11; 1.21.3 Requires-Python >=3.7,<3.11; 1.21.4 Requires-Python >=3.7,<3.11; 1.21.5 Requires-Python >=3.7,<3.11; 1.21.6 Requires-Python >=3.7,<3.11
ERROR: Could not find a version that satisfies the requirement prettytable<4.0.0,>=3.10.1 (from versions: 0.3, 0.4, 0.5, 0.6, 0.6.1, 0.7.1, 0.7.2, 1.0.0, 1.0.1, 2.0.0, 2.1.0, 2.2.0, 2.2.1, 2.3.0, 2.4.0, 2.5.0, 3.0.0, 3.1.0, 3.1.1, 3.2.0, 3.3.0, 3.4.0, 3.4.1, 3.5.0, 3.6.0, 3.7.0, 3.8.0, 3.9.0, 3.10.0)```

I am going to set ==3.10.0 or now unless someone says otherwise